recipes-core/udev/udev/0001-ni-skip-kmod-for-wl12xx.patch

```diff
@@ -4,7 +4,7 @@ index 8551f47..ac8bb60 100644
 +++ b/rules/80-drivers.rules
 @@ -2,6 +2,9 @@
  
- ACTION=="remove", GOTO="drivers_end"
+ ACTION!="add", GOTO="drivers_end"
  
 +# skip kmod for wl12xx device to prevent 30s delay issues with 3.2 kernel
 +ENV{MODALIAS}=="platform:wl12xx", GOTO="drivers_end"
```

Comment:
I think this patch can be dropped, in its entirety, from the Dunfell release.

Also, you can close [this work item](https://dev.azure.com/ni/DevCentral/_workitems/edit/1066354) while you're at it. :)

Reply:
Done.